src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partner = (sex === 'f')
+    ? 'husband\'s'
+    : 'wife\'s';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age ? (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      ) : null}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`My ${partner} name is ${partnerName}`}
+        </p>
+      ) : 'I am not married'}
```

Comment:
`I am not married text should be inside `p` tag, too, move your condition inside `<p className="Person__partner">` tag